lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -16,6 +19,28 @@ def _cached_get_afm_from_fname(fname):
         return AFM(fh)
 
 
+def getSubset(fontfile, characters):
+    """
+    Subset a TTF font
+
+    Reads the named fontfile and restricts the font to the characters.
+    Returns a serialization of the subset font as file-like object.
```

Comment:
It's not public, but you should still try to use numpydoc.

Reply:
I added `Parameters` section, if that's what you meant?

Reply:
Yes, that will work.